spec/MongoStorageAdapter.spec.js

```diff
@@ -390,6 +390,77 @@ describe_only_db('mongo')('MongoStorageAdapter', () => {
     await expectAsync(adapter.getClass('UnknownClass')).toBeRejectedWith(undefined);
   });
 
+
+  /**
+   * If we use equalTo to comparse the nested pointer it works
+   * But it does not work with contained in or matchesQuery
+   */
+  it('queries nested objects using equalTo', async () => {
```

Comment:
I doesn't seem like these tests belong in the `MongoStorageAdapter.spec.js` file, which is testing the adapter itself. These are just Parse.Query tests and there should be a dedicated file for that.